db/migrate/20230728223332_add_status_to_merchants.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToMerchants < ActiveRecord::Migration[7.0]
+  def change
+    add_column :merchants, :status, :integer, default: 0
```

Comment:
Do we want to make the default status for merchants enabled (1) instead of disabled (0)?

If all Merchants are disabled by default, will that impact future functionality and user stories?

Reply:
a segment from US_29: 

When I fill out the form I click ‘Submit’
Then I am taken back to the admin merchants index page
And I see the merchant I just created displayed
_And I see my merchant was created with a default status of disabled._

Reply:
My b!